Update contact.md (#1226)
xsf/xmpp.org: content/contact.md

    - "/contact.html"
---

There are a number of ways to contact the XMPP Standards Foundation or its members.

> - **General inquiry**: If you have a general enquiry please email *info (at) xmpp (dot) org*. If you’d prefer not to talk to an anonymous email address, many of the [XSF Board and XMPP Council members](/about/xmpp-standards-foundation) can be contacted using the details in their profiles.

Reviewer: ```suggestion
- **General enquiry**: For general enquiries please email *info (at) xmpp (dot) org*. If you prefer to contact someone directly, many of the [XSF Board and XMPP Council members](/about/xmpp-standards-foundation) list contact details in their profiles.
```

> - **Chat & mailing lists**: Finally you can join in the many [XMPP Discussions](/community/chat) occurring on the mailing lists.

Reviewer: ```suggestion
- **Chat & mailing lists**: Finally, you can join one of many [XMPP-related discussions](/community/chat) by chat or by using the mailing lists.
```

> - **Sponsorship opportuntities**: If you want to enquire about sponsorship opportuntities please email *sponsors (at) xmpp (dot) org*.

Reviewer: ```suggestion
- **Sponsorship opportunities**: For sponsorship opportunities please email *sponsors (at) xmpp (dot) org*.
```
